src/lib/Components/Bidding/Screens/ConfirmBid.tsx

```diff
@@ -135,13 +135,12 @@ export class ConfirmBid extends React.Component<ConfirmBidProps, ConformBidState
   verifyBidPosition(results, errors) {
     // TODO: Need to handle if the results object is empty, for example if errors occurred and no request was made
     // TODO: add analytics for errors
-    const status = results.createBidderPosition.result.status
+    const { status, position, message_header, message_description_md } = results.createBidderPosition.result
```

Comment:
As long as we're refactoring this, the comment on line 136 about the `results` object being `null` could be looked at. No pressure though.

Reply:
I just created a new story to address the error case: https://artsyproduct.atlassian.net/browse/PURCHASE-187